sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -88,10 +88,12 @@
     "@azure-rest/core-client-paging": "1.0.0-beta.1",
     "@azure-rest/core-client": "1.0.0-beta.6",
     "@azure/core-rest-pipeline": "^1.1.0",
+    "@azure-rest/core-client-lro": "1.0.0-beta.1",
```

Comment:
I deleted `getPoller` so I think we can move this dep to `devDependencies` now unless you want to add `getPoller` back but I do not see the point for it after the simplified `getLongRunningPoller`.

Reply:
Discussed offline. Brought back `getPoller`, see comment above for details